functions/Restore-DbaDatabase.ps1

```diff
@@ -221,8 +221,30 @@ folder for those file types as defined on the target instance.
     }
     PROCESS
     {
-        
-        foreach ($f in $path)
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
+		else
+		{
+			if ($Path.StartsWith("\\") -eq $false)
+			{
+				# Many internal functions parse using Get-ChildItem. 
+				# We need to use Test-SqlPath and other commands instead
+				# Prevent people from trying 
+				
+				Write-Warning "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly."
+				return
+				
```

Comment:
i get how understand works within the continue, but i assuming that the person will pass the same type of path over and over again so it's just sorta a waste to keep trying but maybe that's a bad assumption. changed to continue.

plz plz can i has approval, im so done with this PR